src/telemetry.c

```diff
@@ -15,6 +15,9 @@
  */
 
 #define _GNU_SOURCE
+
+#define EVENT_ID_LEN 32
```

Comment:
Best defined in a single, common header file used in both client and server components.